src/preload.ts

```diff
@@ -101,6 +101,7 @@ const api: IpcApi = {
 		ipcRenderer.invoke( 'getConnectedWpcomSites', localSiteId ),
 	addSyncOperation: ( id: string ) => ipcRenderer.invoke( 'addSyncOperation', id ),
 	clearSyncOperation: ( id: string ) => ipcRenderer.invoke( 'clearSyncOperation', id ),
+	showFilePath: webUtils.getPathForFile,
```

Comment:
```suggestion
	getPathForFile: webUtils.getPathForFile,
```
I don't think `showFilePath` is an ideal name. Something with "get" would be better, since the function is responsible for retrieving a string

Reply:
I was thinking about it too, but decided to take this name, since it was somewhere in documentation, don't remember where exactly I saw it. It's hard to provide some arguments or explain my thoughts, but I will try 😅
This name is provided by creators of Electron, so maybe it will be kinda-ish `following style guides for naming`. I just want to believe, that they were thinking about something, when gave this name to "expose it to main world".
Totally not strong arguments, it's more about my feelings 😅 So I am ok to go with either name, just decided to explain my decision 🙂

Also, if we decided to use something else instead of `showFilePath ` - I think it's better to chose whatever, but not `getPathForFile `, to simplify `Command + F` across our codebase, avoid confusion and simplify reading code at a glance.

Reply:
> if we decided to use something else instead of showFilePath  - I think it's better to chose whatever, but not getPathForFile , to simplify Command + F across our codebase, avoid confusion and simplify reading code at a glance

I actually think that reusing the `getPathForFile` name makes sense since we are simply exposing that API without any modifications.